readthedocs/search/api/__init__.py

```diff
@@ -0,0 +1,102 @@
+from collections import namedtuple
+from math import ceil
+
+from django.utils.translation import gettext as _
+from rest_framework.exceptions import NotFound
+from rest_framework.pagination import PageNumberPagination
+
+
+class PaginatorPage:
```

Comment:
I don't really like putting code under `__init__.py` files. I'd prefer if we put these classes under `readthedocs.search.api.paginators` for example, instead.